.github/workflows/deploy.yml

```diff
@@ -12,22 +12,27 @@ jobs:
     name: Deploy and test
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
-      - uses: actions/setup-python@v5
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set up Python
+        uses: actions/setup-python@v5
         with:
           python-version: 3.12
-          cache: pip
-      - name: Install Modal
-        run: pip install -r requirements.txt
+      - name: Install Poetry
+        run: pip install poetry
+      - name: Install dependencies
+        run: poetry install
       - name: Deploy to dev
-        run: modal deploy tools/deploy.py
+        run: poetry run chemenv deploy
```

Comment:
Can we try to use `uv` instead?

Reply:
Optional - just we've found it to speed up build times quite a bit

Reply:
sure, I'm also moving ChemBench to `uv` atm